sdks/wasm/runtime-tests.js

```diff
@@ -283,6 +284,9 @@ var App = {
 					print ("Exception:" + string_get_utf8 (res));
```

Comment:
this is weird, res is either an int or a utf8 char*?

Reply:
Its a MonoObject which is either an int or a string, see mono_wasm_invoke_method (), it is a bit weird.